app/models/match.rb

```diff
@@ -9,6 +9,7 @@ class Match < ApplicationRecord
   scope :won_home, -> { finished.where(result: 'home') }
   scope :won_away, -> { finished.where(result: 'away') }
   scope :draw, -> { finished.where(result: 'draw') }
+  scope :played, -> { where('date < :now', now: Time.zone.now) }
```

Comment:
Since Rails 6, the convention is to avoid using SQL strings in the queries and rely on the support for time ranges:

```
irb(main):006> puts Match.where(date: ...Time.zone.now).to_sql
SELECT "matches".* FROM "matches" WHERE "matches"."date" < '2023-11-01 17:18:26.277715'
=> nil
```

```suggestion
  scope :played, -> { where(date: ...Time.zone.now) }
```

Reply:
More magic coming along :smile:  and another interesting method `.to_sql`.